All_Relationships/3.data-gen-model.py

```diff
@@ -0,0 +1,136 @@
+
+# coding: utf-8
+
+# # MUST RUN AT THE START OF EVERYTHING
```

Comment:
Is this in reference to the first cell only? It's formatted as the title for the entire notebook. I'd recommend titling the notebook by what steps it performs in the overall analysis pipeline.